hud-data/goog-hud.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "googHUD",
+  "author": "grey",
+  "contributors": [
+    "grey"
+  ],
+  "releaseDate": "9/21/2023",
+  "flags": [
+    "menus",
+    "scoreboards",
+    "materials"
+  ],
+  "tags": [
+    "minimal",
+    "clean",
+    "competitive",
+    "comp",
+    "wagoogus",
+    "beta",
+    ""
+  ],
+  "repo": "https://github.com/GreySux/googhud-main",
+  "hash": "00d40765d044b014651d93cfd28db2780b9069fc",
+  "prerelease": true,
+  "resources": [
+    "https://www.youtube.com/watch?v=C4lgL-YI43U",
+    "cibahud",
+    "voidhud",
+    "cutehud"
+  ],
+  "social": {
+    "support": "https://paypal.me/greysux?country.x=US&locale.x=en_US",
+    "steam_profile": "76561198050618142",
+    "twitter": "https://twitter.com/Grey_Sucks",
+    "discord": "greysucks",
```

Comment:
Must be a discord server invite link, not a personal account. If you don't have a discord server and prefer people reaching out to DMs, I can look into adding support for this

Reply:
Hi! Yes, I would prefer people reaching out in DMs, if that is something you can add.